### PR TITLE
Bump to 3.52.0

### PR DESCRIPTION
**Pull request by aparcar**

### CODE REVIEW

**Comment by aparcar** [comment]
Anything blocking this?

**Comment by sergejkozin** [comment]
@dtkav, could you please review/merge this PR?

File: swagger_ui_bundle/vendor/swagger-ui-3.52.0/index.html [hunk 1]
```diff
@@ -33,13 +33,13 @@
...
-    <script src="./swagger-ui-bundle.js"> </script>
-    <script src="./swagger-ui-standalone-preset.js"> </script>
+    <script src="./swagger-ui-bundle.js" charset="UTF-8"> </script>
+    <script src="./swagger-ui-standalone-preset.js" charset="UTF-8"> </script>
     <script>
     window.onload = function() {
       // Begin Swagger UI call region
       const ui = SwaggerUIBundle({
-        url: "https://petstore.swagger.io/v2/swagger.json",
+        url: "http://localhost:5000/openapi.json",
```
Comment (sergejkozin) [inline swagger_ui_bundle/vendor/swagger-ui-3.52.0/index.html]: @aparcar, I think it better to revert the standard URL in this place?
`https://petstore.swagger.io/v2/swagger.json` is more general for all users of `swagger_ui_bundle`.

Comment (aparcar) [reply swagger_ui_bundle/vendor/swagger-ui-3.52.0/index.html]: Updated.

**Comment by dtkav** [comment]
Thanks for the PR -
It ended up being easier to pull in changes directly rather than review each of the files here.
Feel free to take a look at master and let me know if I you see anything unexpected.

**Pull request closed without merging**